blazor/datepicker/date-format.md

```diff
@@ -11,22 +11,18 @@ documentation: ug
 
 Date format is a way of representing the date value in different string formats in text box.
```

Comment:
Date format is a way of representing the date value in different string formats in DatePicker control.


**Check and include below contents**

- _The date format can be used to specify how the date value is displayed or entered in a DatePicker control_

- _The string format of the date value specifies how the date value should be represented as a string. Different countries and regions have different conventions for representing the date value in a string format. For example, in the United States, the common date format is month/day/year (e.g., "12/28/2022"), while in many other countries, the day/month/year format is more common (e.g., "28/12/2022"). In addition to representing the date value in different string formats, it is also possible to specify the order in which the day, month, and year values appear in the string. For example, the day/month/year format could be written as "28-12-2022" or "28.12.2022"_